src/libraries/System.Data.Common/tests/System/Data/Common/DbProviderFactoriesTests.cs

```diff
@@ -69,6 +70,7 @@ public void GetFactoryWithDataRowTest()
         public void RegisterFactoryWithTypeNameTest()
         {
             ClearRegisteredFactories();
+            Console.WriteLine("Name: " + typeof(System.Data.SqlClient.SqlClientFactory).AssemblyQualifiedName);
```

Comment:
Remove? Assuming this was for debugging the test.

Reply:
:-)